packages/strapi-utils/lib/__tests__/content-types.test.js

```diff
@@ -98,34 +98,15 @@ describe('Content types utils', () => {
   });
 
   describe('getVisibleAttributes', () => {
-    test('Excludes published_at', () => {
+    test('Excludes non visible fields', () => {
       const model = createModel({
         attributes: {
           title: {
             type: 'string',
           },
           [constants.PUBLISHED_AT_ATTRIBUTE]: {
```

Comment:
I don't think it makes sense to use `constants.PUBLISHED_AT_ATTRIBUTE` anymore. Can you put a random attribute name instead?